csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.Gpu.props

```diff
@@ -26,7 +26,8 @@
       <Visible>false</Visible>
     </None>
     <None Include="$(MSBuildThisFileDirectory)..\..\runtimes\win-x64\native\mkldnn.dll"
-          Condition="'$(PlatformTarget)' == 'x64'">
+          Condition="'$(PlatformTarget)' == 'x64' AND
+                     Exists('$(MSBuildThisFileDirectory)..\..\runtimes\win-x64\native\mkldnn.dll')">
```

Comment:
mkldnn is not part of this release.

Reply:
yes, that's fine -- this will only look for the dll if it exists (i.e. just future-proofing). The CPU props file is the same (if it's available, it'll copy it, otherwise this statement is ignored)